sendwp/handler.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+add_action( 'wp_ajax_caldera_forms_sendwp_remote_install', 'wp_ajax_caldera_forms_sendwp_remote_install_handler' );
```

Comment:
Could you comment the function or purpose of the action ?

Reply:
I added a README file for the SendWP SDK usage.